src/tests.builds

```diff
@@ -5,7 +5,7 @@
   <Import Project="$(ToolsDir)ConstructSharedFx.targets" Condition="Exists('$(ToolsDir)ConstructSharedFx.targets')" />
 
   <ItemGroup>
-    <Project Include="$(MSBuildThisFileDirectory)*\tests\**\*.csproj" Exclude="@(TestProjectExclusions)">
+    <Project Include="$(MSBuildThisFileDirectory)*\tests\**\*.Tests.csproj" Exclude="@(TestProjectExclusions)">
```

Comment:
It's the same as before, in essence. If you have a better suggestion then we can consider it.